gw_eccentricity/load_data.py

```diff
@@ -524,30 +534,33 @@ def load_lvcnr_waveform(**kwargs):
         returned.
 
     include_zero_ecc: bool
-        If True returns PhenomT waveform mode for same set of parameters
+        If True, returns PhenomT waveform mode for the same set of parameters
```

Comment:
PhenomT should not be hard coded like this, right?
Please make sure there are no other inconsistencies of any kind.

Reply:
Done